css/index.css

```
@@ -1,3 +1,456 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+}
+
+p {
+	line-height: 1.15;
+}
+
+header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	/* border: 1px solid red; */
+}
+
+nav {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	/* border: 1px solid blue; */
+}
+
+nav a {
+	padding-right: 65px;
+	color: darkgray;
+	/* font-size: 90%; */
+	text-decoration: none;
+	padding-top: 5px;
+}
+
+.top {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	margin-top: 70px;
+}
+
+.innovation {
+	width: 400px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	padding-left: 150px;
+}
+
+.innovation h1 {
+	font-size: 75px;
+	white-space: pre-line;
+	text-align: center;
+}
+
+.button {
+	border: 1px solid black;
+	padding: 7px 40px;
+	width: 170px;
+}
+
+hr {
+	border-color: black;
+	margin-top: 70px;
```

Try to avoid using margin-(direction) if you're specifying more than one direction. Same for padding :)
```suggestion
	margin: 70px 0 20px;
```